invenio_records_resources/services/files/service.py

```diff
@@ -48,12 +48,12 @@ def check_permission(self, identity, action_name, **kwargs):
         action_name = self.config.permission_action_prefix + action_name
         return super().check_permission(identity, action_name, **kwargs)
 
-    def get_record(self, id_, identity, action):
+    def get_record(self, id_, identity, action, **kwargs):
```

Comment:
passing kwargs (`file_key`) is unavoidable if we want fine grained file permissions (i.e. depending on file attributes)

Reply:
Agreed, does not seem to be used anywhere else (e.g. outside of services, where it would be okay to be `_`ed)